src/criteria/safe-to-deploy

```diff
@@ -0,0 +1,14 @@
+This crate will not introduce a serious security vulnerability to production
```

Comment:
Can we properly mark these as .txt files?

Reply:
Done.